src/DelegationManagement.sol

```diff
@@ -1,183 +1,317 @@
 // SPDX-License-Identifier: MIT
 
-/** 
+/**
  *
- *  @title: Delegation Management Contract   
+ *  @title: Delegation Management Contract
  *  @date: 21-Dec-2022 @ 10:30
- *  @version: 4.23 
+ *  @version: 4.23
  *  @notes: This is a experimental contract for delegation registry
  *  @author: skynet2030 (skyn3t2030)
  *
  */
 
-pragma solidity >=0.6.0 <=0.8.0;
+pragma solidity 0.8.17;
```

Comment:
Break this change into a different commit. You're "just" not packing the struct differently. You are also changing the compiler to the latest version. This should be a separate commit

Reply:
Sure. Done!